lemarche/www/pages/views.py

```diff
@@ -342,7 +342,7 @@ def csrf_failure(request, reason=""):  # noqa C901
                         tender_dict[key_cleaned] = list() if value[0] == "" else value
         # get user
         if not request.user.is_authenticated:
-            user = get_or_create_user_from_anonymous_content(tender_dict)
+            user = get_or_create_user_from_anonymous_content(tender_dict, source="TENDER")
```

Comment:
```suggestion
            user = get_or_create_user_from_anonymous_content(tender_dict, source="TENDER_FORM")
```
ou même `User.SOURCE_TENDER_FORM` ?